packages/ui/src/components/avatar/btc-avatar-icon.native.tsx

```diff
@@ -3,8 +3,6 @@ import { Avatar } from './avatar.native';
 
 export function BtcAvatarIcon() {
   return (
-    <Avatar bg="ink.background-primary" p="0">
-      <BtcIcon />
-    </Avatar>
+    <Avatar outlineColor="ink.border-transparent" icon={<BtcIcon width="100%" height="100%" />} />
```

Comment:
Maybe we don't need this file? And can instantiate it on demand?

Reply:
💯 See my [comment here](https://github.com/leather-io/mono/pull/858#discussion_r1950284816)